JSDemos/Demos/DataGrid/BatchUpdateRequest/jQuery/index.js

```diff
@@ -0,0 +1,71 @@
+$(function () {
+    var URL = "https://js.devexpress.com/Demos/Mvc/api/DataGridWebApi";
+
+    $("#gridContainer").dxDataGrid({
+        dataSource: {
+            store: DevExpress.data.AspNet.createStore({
+                key: "OrderID",
+                loadUrl: URL + "/Orders",
+                onBeforeSend: function (method, ajaxOptions) {
+                    ajaxOptions.xhrFields = { withCredentials: true };
+                },
+            }),
+            pushAggregationTimeout: 0
```

Comment:
Remove pushAggregationTimeout because we remove push api usage from this demo. Assign createStore to dataSource directly.